### ISSUE TITLE
Can i sort table row using only odd table row count

### ISSUE THREAD

**Issue by RajuTicvic** [Mottie/tablesorter#1648]
Hi @Mottie !
I want filter table row using only table odd row.
Is it possible?
![screenshot-localhost-8000-2019 03 27-12-36-06](https://user-images.githubusercontent.com/47621807/55056739-03732780-508d-11e9-9cd2-3393cf2c1164.png)

**Comment by Mottie** [comment]
Hi @RajuTicvic!

Please clarify. You need the table to sort by parent row (id, user, activity, etc), but only filter by child row (description, initiated by, running messages)?

If that is the case, then it should be possible (I haven't tested it) using:

* A [custom filter function](https://mottie.github.io/tablesorter/docs/example-widget-filter-custom.html#filter_function_information) which would use the `data.childRowText` content to return a match; but using this method won't allow you to filter any of the parent row content.
* Or, create a [custom filter type](https://mottie.github.io/tablesorter/docs/example-widget-filter-custom-search.html#how_to_add_custom_filter_types) in which a symbol or flag can be used (e.g. `~` for a fuzzy match) to target the child row text. This would allow users to search either main row or child row content. 

  Hmm, I like this idea, I think I may add a new filter type (when I have some time) that'll work like this: `child:Admin` - which would only search the child row content for "Admin". But as I have overly busy and I have a huge backlog of work to catch up on, please don't wait for me to implement this enhancement.

**Comment by RajuTicvic** [comment]
@Mottie thank you for your valuable reply,
Here i attached an image one more try to explain to you
![scr1](https://user-images.githubusercontent.com/47621807/55076841-692ad800-50bc-11e9-844b-3e9cdcb359f7.png)

**Comment by Mottie** [comment]
Yes, since you were using child rows, I thought the [cssChildRow](https://mottie.github.io/tablesorter/docs/#csschildrow) class was already added to all the even rows. See [this demo](https://mottie.github.io/tablesorter/docs/example-child-rows-filtered.html)

**Comment by RajuTicvic** [comment]
@Mottie 
i hope this will work, i will use this & let you know

**Comment by RajuTicvic** [comment]
@Mottie can you give me a demo on jsfiddle?

**Comment by Mottie** [comment]
http://jsfiddle.net/Mottie/sdsobvp9/

**Comment by RajuTicvic** [comment]
@Mottie  thank you
